apps/hash-frontend/src/pages/[shortname]/types/entity-type/[entity-type-id].page/use-entity-type-value.tsx

```diff
@@ -192,9 +194,7 @@ export const useEntityTypeValue = (
 
       const newUrl = extractBaseUri(res.data.schema.$id);
 
-      if (newUrl) {
-        await router.replace(newUrl, newUrl, { shallow: true });
-      }
+      await router.replace(newUrl, newUrl, { shallow: true });
```

Comment:
The null/undefined check was removed from here. Are we sure that this would be valid always?

Reply:
I believe so, because `extractBaseUri` will throw an error if it's invalid (which we do not catch or handle). So either way `newUrl` should be valid here